adhocracy4/static/Modal.jsx

```diff
@@ -3,55 +3,69 @@ import django from 'django'
 
 const Modal = (props) => {
   const dismiss = props.dismissOnSubmit ? 'modal' : 'false'
+  const abortStr = props.abort || django.gettext('Abort')
+  const bodyCssClass = props.partials.bodyClass || ''
+
   return (
     <div
-      className="modal fade" id={props.name} tabIndex="-1"
-      role="dialog" aria-labelledby="myModalLabel" aria-hidden="true"
+      className="modal fade"
+      id={props.name}
+      tabIndex="-1"
+      role="dialog"
+      aria-label={props.partials.title || 'Modal'}
+      aria-hidden="true"
     >
-      <div className="modal-dialog modal-lg" role="document">
+      <div
+        className="modal-dialog modal-dialog-centered modal-lg"
+        role="document"
+      >
         <div className="modal-content">
           <div className="modal-header">
+            {props.partials.title && (
+              <h1>
+                {props.partials.title}
+              </h1>)}
             <button
-              className="close" aria-label={props.abort}
-              data-bs-dismiss="modal" onClick={props.handleClose}
+              className="close"
+              aria-label={abortStr}
+              data-bs-dismiss="modal"
+              onClick={props.handleClose}
             >
               <i className="fa fa-times" />
             </button>
           </div>
 
           <div
-            className={'modal-body ' + props.partials.bodyClass}
+            className={'modal-body ' + bodyCssClass}
           >
-            <h3 className="modal-title u-first-heading">{props.partials.title}</h3>
+            <div className="u-spacer-bottom">
+              <span className="modal-title u-first-heading">
+                {props.partials.description}
+              </span>
```

Comment:
why are we are using modal-title class and a h1 partial.title ?

Reply:
right, this span is not necessary...removing it.